prost-types/Cargo.toml

```diff
@@ -16,11 +16,15 @@ doctest = false
 default = ["std"]
 std = ["prost/std"]
 arbitrary = ["dep:arbitrary"]
+serde = ["serde_serialize", "serde_deserialize"]
+serde_serialize = ["std", "dep:serde"]
+serde_deserialize = ["std", "dep:serde"]
```

Comment:
Why is it important to split the feature? The compiler is smart enough to remove unused code from the binary, right? I suggest having a single `serde` feature.